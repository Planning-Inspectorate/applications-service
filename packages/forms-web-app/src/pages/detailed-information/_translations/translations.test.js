const enDetailedInformationTranslations = require('./en.json');
const cyDetailedInformationTranslations = require('./cy.json');

describe('pages/detailed-information/_translations', () => {
	it('should return the correct English translations', () => {
		expect(enDetailedInformationTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh translations', () => {
		expect(cyDetailedInformationTranslations).toMatchSnapshot();
	});

	it('should return the same keys for English and Welsh translations', () => {
		expect(enDetailedInformationTranslations).toHaveSameKeys(cyDetailedInformationTranslations);
	});
});
