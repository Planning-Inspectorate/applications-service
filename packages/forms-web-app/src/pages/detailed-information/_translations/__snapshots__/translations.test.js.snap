// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/detailed-information/_translations should return the correct English translations 1`] = `
{
  "heading1": "Detailed information",
  "linkParagraph1": "View the stages an application goes through, from start to finish.",
  "linkParagraph2": "Find out everything you need to know about sharing your views on a project.",
  "linkParagraph3": "Outlining the application process with resources on submitting a planning proposal.",
  "linkParagraph4": "View the 2008 Planning Act to understand our legal duties.",
  "linkParagraph5": "View documents around submitting applications for a Nationally Significant Infrastructure project.",
  "linkParagraph6": "The Planning Inspectorate has a series of advice notes covering a range of process details.",
  "linkParagraph7": "Find out what National Policy Statements (NPSs) are, what they include and how they fit into the Planning Act 2008 process.",
  "linkParagraph8": "Here you can view all advice provided by the Planning Inspectorate since 2008.",
  "linkTitle1": "See the process",
  "linkTitle2": "How to have your say on a project",
  "linkTitle3": "Guide for applicants",
  "linkTitle4": "Legislation information",
  "linkTitle5": "View guidance",
  "linkTitle6": "See advice notes",
  "linkTitle7": "National Policy Statements",
  "linkTitle8": "Register of advice",
  "paragraph1": "Find further legislation and guidance resources. Information on this page may be useful for those applying to use the service and the general public.",
}
`;

exports[`pages/detailed-information/_translations should return the correct Welsh translations 1`] = `
{
  "heading1": "Gwybodaeth fanwl",
  "linkParagraph1": "Gallwch weld y camau y mae cais yn mynd trwyddynt, o'r dechrau i'r diwedd.",
  "linkParagraph2": "Dysgwch bopeth y mae angen i chi ei wybod am rannu eich safbwyntiau ar brosiect.",
  "linkParagraph3": "Canllaw sy'n amlinellu'r broses ymgeisio gydag adnoddau ar gyflwyno cynnig cynllunio.",
  "linkParagraph4": "Gallwch weld Deddf Cynllunio 2008 i ddeall ein dyletswyddau cyfreithiol.",
  "linkParagraph5": "Gallwch weld dogfennau ynglŷn â chyflwyno ceisiadau ar gyfer Prosiect Seilwaith o Arwyddocâd Cenedlaethol.",
  "linkParagraph6": "Mae gan yr Arolygiaeth Gynllunio gyfres o nodiadau cyngor sy'n ymdrin ag ystod o fanylion proses.",
  "linkParagraph7": "Dysgwch beth yw Datganiadau Polisi Cenedlaethol (NPSau), beth maen nhw'n ei gynnwys a sut maen nhw'n berthnasol i broses Deddf Cynllunio 2008.",
  "linkParagraph8": "Gallwch weld yr holl gyngor a roddwyd gan yr Arolygiaeth Gynllunio ers 2008.",
  "linkTitle1": "Gweld y broses",
  "linkTitle2": "Sut i leisio'ch barn am brosiect",
  "linkTitle3": "Canllaw i ymgeiswyr",
  "linkTitle4": "Gwybodaeth ddeddfwriaethol",
  "linkTitle5": "Gweld canllawiau",
  "linkTitle6": "Gweld nodiadau cyngor",
  "linkTitle7": "Datganiadau Polisi Cenedlaethol",
  "linkTitle8": "Cofrestr gyngor",
  "paragraph1": "Dewch o hyd i ragor o ddeddfwriaeth ac adnoddau canllaw. Gallai'r wybodaeth ar y dudalen hon fod yn ddefnyddiol i'r rhai sy'n gwneud cais i ddefnyddio'r gwasanaeth ac i'r cyhoedd.",
}
`;
